query/autocomplete.js

```diff
@@ -59,23 +61,28 @@ query.filter( peliasQuery.view.boundary_rect );
 **/
 function generateQuery( clean ){
 
-  var vs = new peliasQuery.Vars( defaults );
+  const vs = new peliasQuery.Vars( defaults );
+
+  let logStr = '[query:autocomplete] [parser:addressit] ';
```

Comment:
minor nitpick: it might make sense to rename this something like `paramLogStr`, since we could be logging lots of things in this file.